lib/LaTeXML/Core/Tokens.pm

```diff
@@ -84,13 +100,24 @@ sub beDigested {
 
 sub neutralize {
   my ($self, @extraspecials) = @_;
-  return Tokens(map { $_->neutralize(@extraspecials) } $self->unlist); }
+  my @neutralized = map { $_->without_dont_expand } $self->unlist;
+  if (ref @$self[-1] eq 'ARRAY') {    # preserve marks
+    push(@neutralized, @$self[-1]); }
+  return Tokens(map { $_->neutralize(@extraspecials) } @$self); }
+
+sub without_dont_expand {
+  my ($self) = @_;
+  my @cleaned = map { $_->without_dont_expand } $self->unlist;
+  if (ref @$self[-1] eq 'ARRAY') {    # preserve marks
+    push(@cleaned, @$self[-1]); }
+  return Tokens(@cleaned); }
```

Comment:
So, here, you remove the `\noexpand` flag, but keep the `\the` mask/mark?

Reply:
Exactly, the masks are kept, because the two mechanisms are conceptually independent from each other. I don't know if I actually need this extra care in practice, but it made sense to me to add it.